.changeset/dry-actors-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
+"@coinbase/onchainkit": patch
+---
+
+**feat**: `useSendCall` and `useSendCalls` by @0xAlec #1130
```

Comment:
Let's make sure we are a bit more verbose in what those function do.

The goal of a CHANGELOG is to inform and help the developer understand what's going on.

Reply:
done in new diff!